File: Problem/26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums)
    {
        if (nums.empty())
            return (0);
        int j = 1;
        sort(nums.begin(), nums.end());
        for (int i= 1; i < nums.size();i++)
        {
            if (nums[i] != nums[j - 1])
                {
                    nums[j] = nums[i];
                    j++;
                }
        }
        return (j);

    }
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        iter = 1
        index = 1
        while (iter < len(nums)):
            if (nums[index-1] != nums[iter]):
                nums[index] = nums[iter]
                index +=1
            iter+=1
        return (index)
        